Ops/src/OpsWorkerGenetic.cpp

```diff
@@ -71,30 +74,23 @@ void WorkerGenetic::run()
     mIterationCounter=0;
     for(; mIterationCounter<mnMaxIterations && !mpMessageHandler->aborted(); ++mIterationCounter)
     {
-        for(size_t i=0; i<mNumCandidates; i+=2) {
+        mCandidatePoints.clear();
+        std::vector<size_t> ids = getIdsSortedFromWorstToBest();
+        for(int i=0; i<mNumElites; ++i) {
+            mCandidatePoints.push_back(mPoints[ids.back()]);
+            ids.pop_back();
+        }
+        while(mCandidatePoints.size() < mNumPoints) {
```

Comment:
When selecting parents for crossover, will the elites be used again ? And if that is that case it is a problem or maybe as it should be? I dont remember. Maybe there is a risk for the elites to have to much "say" since both they them selves and their crossover-children will survive in this case ? OR maybe this is indented ?

Reply:
Aha, just noticed that parents are chosen randomly, so maybe not an issue.

Reply:
There are different ways to implement elitism, but when reading about it online, it seems like the most common approach is that elites should also be able to be selected as parents. There are also different opinions on whether or not the elites should undergo mutation or not. But to begin with we should just stick to the most simple approach I think.